core/templates/pages/moderator-page/moderator-page.component.html

```diff
@@ -1,8 +1,8 @@
 <div>
   <div class="oppia-content">
     <br>
-    <ul ngbNav #nav="ngbNav" [animation]="false">
-      <li [ngbNavItem]="1">
+    <nav ngbNav #nav="ngbNav" [animation]="false">
```

Comment:
These changes were to fix the accessibility rule "List items (<li>) are not contained within <ul> or <ol> parent elements." and get the moderator page to 100%. Without it, lighthouse build fails.

See report from the commit before this change: https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1628902523695-84265.report.html#accessibility